azure-container-apps-arm/aca-op-scim-bridge-template.json

```
@@ -0,0 +1,252 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "variables": {
+        "resourceSettings": {
+            "base": {
+                "cpu": "0.25",
+                "memory": "0.5Gi"
+            },
+            "high": {
+                "cpu": "0.5",
+                "memory": "1Gi"
+            },
+            "very high": {
+                "cpu": "1",
+                "memory": "2Gi"
+            }
+        }
+    },
+    "parameters": {
+        "containerAppName": {
+            "defaultValue": "op-scim-con-app",
+            "type": "String",
+            "metadata": {
+                "description": "The name of the container app."
+            }
+        },
+        "containerAppEnvName": {
+            "defaultValue": "op-scim-con-app-env",
+            "type": "String",
+            "metadata": {
+                "description": "The name of the container app environment."
+            }
+        },
+        "logAnalyticsWorkspaceName": {
+            "defaultValue": "op-scim-con-app-log-analytics",
+            "type": "String",
+            "metadata": {
+                "description": "The name of the log analytics workspace."
+            }
+        },
+        "scimsession": {
+            "type": "SecureString",
+            "metadata": {
+                "description": "The content of your scimsession file in plain text"
```

Missing punctuation:

```suggestion
                "description": "The content of your scimsession file in plain text."
```

I might also word it _slightly_ differently, i.e.:

```suggestion
                "description": "The plain text contents of your scimsession file."
```